### PR TITLE
Non-magic non-special projectiles become four times faster

### PR DESCRIPTION
**Pull request by ghost**
[Changelogs]: testmerge if possible
:cl: zamolxius
balance: rebalanced projectiles, except magic and special ones, to be four times faster
/:cl:

[why]:  makes projectiles less trivial and may change the meta. testmerge if possible

### CODE REVIEW

**Review by yoyobatty — COMMENTED** [review]
I was going to make another pr that doubled shotgun pellets but halved damage

**Comment by RafRoq** [comment]
+1

**Comment by yoyobatty** [comment]
This doesn't affect energy weapons at all which still travel at a speed of 0.7

**Comment by RafRoq** [comment]
Then make energy weapons travel faster too

**Comment by ghost** [comment]
ok lemme see

**Comment by ghost** [comment]
@t-f-w @Lxtoofonea @bardofrage

**Comment by bardofrage** [comment]
+1, no reason to currently use a gun over a taser.

**Comment by earthmoonjune** [comment]
+1

**Comment by Manly-Dwarf** [comment]
Does this speed up tasers and disablers?
If so, the headmins just +1'd this and we're all doomed. I hate you.

File: hippiestation/code/modules/projectiles/projectile.dm [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+/obj/item/projectile/special
+	speed = 0.8 //as before
+
+/obj/item/projectile/energy
+	speed = 0.2
+
+/obj/item/projectile/beam
+	speed = 0.2
+
+/obj/item/projectile/bullet/p50
```
Comment (Manly-Dwarf) [inline hippiestation/code/modules/projectiles/projectile.dm:19]: Uhh?

**Comment by earthmoonjune** [comment]
:agony:

File: hippiestation/code/modules/projectiles/projectile.dm [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/obj/item/projectile
+	speed = 0.2 //four times faster
+
+/obj/item/projectile/magic
```
Comment (Manly-Dwarf) [inline hippiestation/code/modules/projectiles/projectile.dm:4]: If you're going to be specific with p90, why not be specific with fireball & MM?

**Comment by yoyobatty** [comment]
Indeed it does.

**Review by Manly-Dwarf — CHANGES_REQUESTED** [review]

File: hippiestation/code/modules/projectiles/projectile.dm (hunk 2 above)
Comment (yoyobatty) [reply hippiestation/code/modules/projectiles/projectile.dm:4]: P50 is actually a custom speed of `0.4`, I codechecked.

**Comment by earthmoonjune** [comment]
don’t speed those up you FOOL

File: hippiestation/code/modules/projectiles/projectile.dm (hunk 1 above)
Comment (yoyobatty) [reply hippiestation/code/modules/projectiles/projectile.dm:19]: For disablers, pulse rifles, lethal eguns

Comment (Manly-Dwarf) [reply hippiestation/code/modules/projectiles/projectile.dm:19]: ABORT
Also what this was set to was p50, was wondering why it was here

**Review by yoyobatty — CHANGES_REQUESTED** [review]
Please ammend changes

**Comment by bardofrage** [comment]
Gonna have to retract my -1, unless the taser remains at normal speed.

**Comment by ghost** [comment]
how about taser just a mite faster?

File: hippiestation/code/modules/projectiles/projectile.dm (hunk 1 above)
Comment (ghost) [reply hippiestation/code/modules/projectiles/projectile.dm:19]: p50 is the sniper rifle bullet iirc

File: hippiestation/code/modules/projectiles/projectile.dm (hunk 2 above)
Comment (ghost) [reply hippiestation/code/modules/projectiles/projectile.dm:4]: magic isnt propulsed by the holy power of bullet propellants and the Omnissiah

File: hippiestation/code/modules/projectiles/projectile.dm [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+/obj/item/projectile/special
+	speed = 0.8 //as before
+
+/obj/item/projectile/energy
+	speed = 0.2
+
+/obj/item/projectile/energy/electrode
+	speed = 0.8
+
+/obj/item/projectile/beam
```
Comment (GuyonBroadway) [inline hippiestation/code/modules/projectiles/projectile.dm:16]: this has been redefined in another file, this means we will have a redefine of a redefine which is probably bad.

Comment (yoyobatty) [reply hippiestation/code/modules/projectiles/projectile.dm:16]: Yes, please test every single one of these in VV. You can check by spawning ammo_casing and then checking the reference to the projectile.

Comment (ghost) [reply hippiestation/code/modules/projectiles/projectile.dm:16]: disabler was redefined as 0.7 but i deleted that

**Review by yoyobatty — APPROVED** [review]
Okay, looks fine has enough +1's and followed requested changes, going to merge.

**Pull request merged**